src/domain.jl

```diff
@@ -100,11 +91,11 @@ function affect!(integrator, f::AbstractDomainAffect{T,S,uType}) where {T,S,uTyp
 
     # update current and next time step
     if dt_modified # add safety factor since guess is based on extrapolation
-        set_proposed_dt!(integrator, 9//10*integrator.dt)
+        set_proposed_dt!(integrator, 9//10*dt)
+        change_t_via_interpolation!(integrator,t)
```

Comment:
Do we want this? Isn't setting `dtproposed` sufficient?

Reply:
Yeah I forgot what exactly this technique was trying. It wasn't a pullback rejection via the interpolation, though such a method would be useful.